apps/component-date/index.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+const summary = require('hof-behaviour-summary-page');
+
+module.exports = {
+  name: 'component-date',
+  baseUrl: '/component-date',
```

Comment:
Ack, you are right. However I think that's more to do with the repo/module naming convention we have of `hof-<module-type>-<name>` than anything else. I still think `date-component` reads better and is more user friendly.

Also, if you're going to think of me as Yoda then I'm going to have to insist on being able to claim piggy-back rides.